pkg/api/v1/users.go

```diff
@@ -0,0 +1,186 @@
+package api
+
+import (
+	"database/sql"
+	"net/mail"
+	"time"
+
+	"go.rtnl.ai/quarterdeck/pkg/store/models"
+	"go.rtnl.ai/ulid"
+)
+
+type User struct {
+	ID          ulid.ULID `json:"id,omitempty"`
+	Name        string    `json:"name,omitempty"`
+	Email       string    `json:"email"`
+	LastLogin   time.Time `json:"last_login,omitempty"`
+	Roles       []*Role   `json:"roles"`
```

Comment:
Would a user need to have more than 1 role/title? Also, will users need to have a `role` and a `title`. For example, a user could have an `admin` role in the app and a `CEO` title. This may be a question for Ben since I know that the User model makes it possible for a user to have multiple roles.